### ISSUE TITLE
aplcms reporting HDF5 error: all chunk dimensions must be positive

### ISSUE THREAD

**Issue by martenson** [RECETOX/recetox-aplcms-archive#1]
This is on the execution of 570 "positive" samples.

```
Error in dataset_create_pl$set_chunk(chunk_dims) : HDF5-API Errors:
    error #000: ../../../src/H5Pdcpl.c in H5Pset_chunk(): line 2036: all chunk dimensions must be positive
        class: HDF5
        major: Invalid arguments to routine
        minor: Out of range
Calls: <Anonymous> ... [[<- -> [[<-.H5File -> <Anonymous> -> <Anonymous> -> .Call
Execution halted
```

**Comment by xtracko** [comment]
The memory and dimensionality analysis of an h5 output on 100, 200 and 300 input files. I've focused my attention on 3 contained tables `peaks`, `extracted_features` and `corrected_features`. The tables `extracted_features` and `corrected_features` should be the biggest, and they are produced by concatenating extracted features from all the files. Thus, they may exceed the R limit of `2^31` rows.

| num_of_mzml_files | peaks_nrow | peaks_ncols | peaks_mem | ext_feat_nrow | ext_feat_ncol | ext_feat_mem | cor_feat_nrow | cor_feat_ncol | cor_feat_mem |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 100 | 40846 | 204 | 0.1 GiB | 2490885 | 5 | 0.1 GiB | 2490885 | 7 | 0.1 GiB |
| 200 | 45310 | 404 | 0.1 GiB | 5099283 | 5 | 0.2 GiB | 5099283 | 7 | 0.3 GiB |
| 300 | 48280 | 604 | 0.2 GiB | 8045846 | 5 | 0.3 GiB | 8045846 | 7 | 0.4 GiB |

This seems to scale "just" fine. On 300 files, the number of rows is just under `2^23`, i.e. far from the R's limit of `2^31` and 400 files should succeed. 

I will continue my investigation with a more thorough analysis of the failed 400 and 500 file runs.

**Comment by hechth** [comment]
@martenson @xtracko Can this be closed after having merged #14 ?

**Comment by martenson** [comment]
yeah, hopefully this ends up as #wontfix and does not bite us with parquet
